src/main/groovy/com/morpheusdata/hyperv/HyperVApiService.groovy

```diff
@@ -0,0 +1,1157 @@
+package com.morpheusdata.hyperv
+
+import com.morpheusdata.core.MorpheusContext
+import groovy.json.JsonOutput
+import groovy.util.logging.Slf4j
+
+/**
+ * @author rahul.ray
+ */
+
+@Slf4j
+class HyperVApiService {
+
+    MorpheusContext morpheusContext
+
+    HyperVApiService(MorpheusContext morpheusContext) {
+        this.morpheusContext = morpheusContext
+    }
+
+    static defaultRoot = 'C:\\morpheus'
+
+    def executeCommand(command, opts) {
+        def output
+        //= morpheusContext.executeWindowsCommand(address, port, username, password, command, noProfile, elevated)
```

Comment:
This needs to be implemented before this PR can be approved.

Reply:
@ddeyoung review comment changes added